tests/app/views/contexts/conftest.py

```diff
@@ -0,0 +1,53 @@
+import json
+from mock import MagicMock
+from pathlib import Path
+
+import pytest
+
+from app.data_model.answer import Answer
+from app.data_model.answer_store import AnswerStore
+from app.data_model.list_store import ListStore
+from app.forms.questionnaire_form import QuestionnaireForm
+from app.setup import create_app
+
+
+@pytest.fixture
+def rendered_block():
+    path = Path(__file__).parent.joinpath('test_list_collector_rendered_block.json')
```

Comment:
Loading a big block of JSON like this is something we've tried to avoid in the unit tests where possible. 

Is there a way we can reduce this and keep it in python as has been done elsewhere?

Reply:
Can do - are you suggesting adding it as a dict to conftest?

Reply:
Potentially, I guess the ideal would be if we could make a more minimal example that could be represented as a dict.

@iwootten Did make a good point in that when reading the test, you normally need to look at the fixtures as well. So unless there are multiple test files sharing the fixtures it might be worth moving them into the test file for the moment. That way we don't end up with a load of apparently common functionality which isn't actually common.